pkg/services/ngalert/schedule/registry.go

```diff
@@ -140,7 +140,11 @@ func (a *alertRuleInfo) update(lastVersion ruleVersion) bool {
 
 type evaluation struct {
 	scheduledAt time.Time
-	rule        *models.AlertRule
+	evaluationData
+}
+
+type evaluationData struct {
```

Comment:
What is going to be in the struct at the end of this series of changes?

Reply:
At the end this struct will contain folder title, AlertRuleGroupKey (probably), and a slice of rules (a group)